babel/src/test/resources/sql/big-query.iq

```diff
@@ -1594,13 +1594,13 @@ WITH letters AS
   SELECT 'b c d' as letter_group)
 SELECT SPLIT(letter_group, ' ') as example
 FROM letters;
-+-----------+
-| example   |
-+-----------+
-| []        |
-| [a]       |
-| [b, c, d] |
-+-----------+
++--------------+
```

Comment:
The test results here look a little strange, but this seems to be as expected. [PR-4024] Changed raggedUnionTypesToVarying, which will fix the problem